cetav-ddw/lalibertad-cetav
# navigation

## Description

The navigation component for the page was created using Chakra-recommended tags for navigation. Additionally, folders for the home and frequently asked questions were created along with their respective page.js files, referencing this page's structure.

### How to verify this PR

1. Verify the navigation component has been implemented using recommended Chakra tags.
2. Check that the home and frequently asked questions folders, along with their respective page.js files, have been successfully created.
3. Confirm that the navigation component references this page's structure correctly.
